qa/tasks/caasp.py

```diff
@@ -0,0 +1,137 @@
+'''
+Task that deploys a CAASP cluster on all the nodes
+Linter:
+    flake8 --max-line-length=100
+'''
+import logging
+import os
+import subprocess
+from util import remote_exec
+from teuthology.exceptions import ConfigError
+from teuthology.misc import (
+    delete_file,
+    move_file,
+    sh,
+    sudo_write_file,
+    write_file,
+    copy_file,
+    all_roles_of_type
+    )
```

Comment:
I suggest to do only
```
from teuthology import misc
```
And later just use this method from the corresponding namespace, like: `misc.delete_files`, `misc.sh`, etc.